streaming/BUILD.bazel

```diff
@@ -276,6 +276,7 @@ genrule(
         mkdir -p "$$GENERATED_DIR"
         touch "$$GENERATED_DIR/__init__.py"
         sed -i -E 's/from streaming.src.protobuf/from ./' "$$GENERATED_DIR/remote_call_pb2.py"
+        sed -i -E 's/from protobuf/from ./' "$$GENERATED_DIR/remote_call_pb2.py"
```

Comment:
Could you add a comment explaining what the purpose of this line and the above line is?

Reply:
added